gum/backend-darwin/gumcodesegment-darwin.c

```diff
@@ -298,6 +306,29 @@ gum_code_segment_try_map (GumCodeSegment * self,
   return result != MAP_FAILED;
 }
 
+static gboolean
+gum_code_segment_try_remap (GumCodeSegment * self,
+                            gsize source_offset,
+                            gsize source_size,
+                            gpointer target_address)
+{
+  mach_vm_address_t address = (mach_vm_address_t) target_address;
+  mach_port_t task_self = mach_task_self ();
```

Comment:
- Let's name this `self_task` for consistency.
- Assignment should be moved below the declarations. (No declaration after statement, for consistency.)